### PR TITLE
Bump crc resources in centos-9-medium-3x-centos-9-crc-extracted-2-36-0-xxl

### PR DESCRIPTION
**Pull request by gibizer**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

**Comment by gibizer** [comment]
We need this resource bump to be able to speed up the upstream test jobs by increasing the openstack service replicas

**Review by SeanMooney — APPROVED** [review]
specifically this will unblock 
https://github.com/openstack-k8s-operators/nova-operator/pull/786

alternitivly we can add a new nodeset but give the time reduction gibi was able to make we will likely want to applied this to other jobs in the future

File: zuul.d/nodeset.yaml [hunk 1]
```diff
@@ -309,7 +309,7 @@
       - name: compute-2
         label: cloud-centos-9-stream-tripleo
       - name: crc
-        label: coreos-crc-extracted-2-36-0-xxl
+        label: coreos-crc-extracted-2-36-0-3xl
```
Comment (viroel) [inline zuul.d/nodeset.yaml:312]: We try to maintain the name in sync with the labels used, in this case, the nodeset name says: `name: centos-9-medium-3x-centos-9-crc-extracted-2-36-0-xxl`. I am in favor of creating a new nodeset with a different name.

Comment (gibizer) [reply zuul.d/nodeset.yaml:335]: done

**Review by lewisdenny — APPROVED** [review]
/approve

**Pull request merged**